### ISSUE TITLE
Issue: Type error for frontMatter.content.sorting data.

### ISSUE THREAD

**Issue by iveptawebadmin** [estruyf/vscode-front-matter#183]
**Describe the bug**

`"Incorrect type. Expected "object".'`

![image](https://user-images.githubusercontent.com/30990811/142817040-9622e2bb-6256-4624-a0af-dfa36569b80d.png)

I think that this is supposed to be an array of objects and not an object itself.

**To Reproduce**

Add this config data to the frontmatter.json file and see the Problem.

```
"frontMatter.content.sorting": [
    {
      "title": "Date (asc)",
      "name": "date",
      "order": "asc",
      "type": "date"
    },
    {
      "title": "Date (desc)",
      "name": "date",
      "order": "desc",
      "type": "date"
    },
    {
      "title": "Title (asc)",
      "name": "title",
      "order": "asc",
      "type": "string"
    },
    {
      "title": "Title (desc)",
      "name": "title",
      "order": "desc",
      "type": "string"
    }
  ],
```

**Expected behavior**

Not show a Problem with the Array of Objects.

**Additional context**

Running on GitPod.io, can share the workspace for troubleshooting if needed.

**Comment by estruyf** [comment]
This has already been fixed for the upcoming release. 😁

**Comment by estruyf** [comment]
Now available in the `v5.6.0` release
